sklearn/metrics/_ranking.py

```diff
@@ -1419,3 +1419,100 @@ def ndcg_score(y_true, y_score, k=None, sample_weight=None, ignore_ties=False):
     _check_dcg_target_type(y_true)
     gain = _ndcg_sample_scores(y_true, y_score, k=k, ignore_ties=ignore_ties)
     return np.average(gain, weights=sample_weight)
+
+
+def top_k_accuracy_score(y_true, y_score, k=2, normalize=True):
```

Comment:
I believe that a reasonable default would be `k=5`.

Reply:
thoughts on https://github.com/scikit-learn/scikit-learn/pull/16625#discussion_r387332185 ?

Reply:
I missed your comment.